notebooks/contrib-dev/Alcohol_Pharmacokinetics.ipynb

```diff
@@ -0,0 +1,1075 @@
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Time to sober up to below the legal limit: 2.10 hours\n"
+          ]
+        }
+      ],
+      "source": [
+        "# You can change tmax(3) here. Increase tmax to 10 h\n",
+        "C = Model_info(2, 75, 10)\n",
+        "\n",
+        "# Add your solution here\n",
+        "### BEGIN SOLUTION\n",
+        "# Plot CA and BAC\n",
+        "fig, ax1 = plt.subplots(figsize=(6.4, 4), dpi=100)\n",
```

Comment:
The font sizes in the axes labels and axes numbers are too small. Please see the instructions for publication quality plots.